### DISCUSSION TITLE
How to enable the github workflow action for the `Error: Resource not accessible by integration`?

### DISCUSSION THREAD

**Discussion by Infinity-chakra** [YSHgroup/vue3-fullcalendar-bootstrap#5] [Q&A] [answered]
Hi @YSHgroup ,
The [create-release.yml](https://github.com/YSHgroup/vue3-fullcalendar-bootstrap/blob/main/.github/workflows/create-release.yml) doesn't seem to work on your side.
I've seen the error
```
Run actions/create-release@v1 
Error: Resource not accessible by integration
```
Could you let me know what case the workflow doesn't work in?

**Comment by YSHgroup** [comment, marked as answer]
Thanks for the question, I've noticed that my settings for the functionality didn't set correctly.
After navigating to `vue3-fullcalendar-bootstrap/settings/actions`, fix the Workflow permissions.
Check the radio button `Read and write permissions` and check box `Allow GitHub Actions to create and approve pull requests`.
And then, work correctly.

**Reply by Infinity-chakra** [reply]
Thank you. it works now.
